embedchain/bots/whatsapp.py

```diff
@@ -3,6 +3,9 @@
 import signal
 import sys
 
+from flask import Flask, request
```

Comment:
we cannot do this.
This is done on purpose.

Reply:
we need to figure out a way to ignore this.
imports needs to be lazily loaded in the __init__

Reply:
Okay will see how it can be done